### PR TITLE
Update language version

### PR DESCRIPTION
**Pull request by kbaley**
Missed some stuff

### CODE REVIEW

File: src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
```
Comment (DavidBoike) [inline src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj:4]: Did you mean to revert this?

File: src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj [hunk 2]
```diff
@@ -12,7 +12,7 @@
   <ItemGroup>
     <PackageReference Include="McMaster.Extensions.CommandLineUtils" Version="4.0.1" />
     <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.7.0" />
-    <PackageReference Include="Particular.Packaging" Version="2.1.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="2.0.0" PrivateAssets="All" />
```
Comment (DavidBoike) [inline src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj:15]: Did you mean to revert this?

File: src/CommandLineTests/NServiceBus.Transport.AzureServiceBus.CommandLine.Tests.csproj [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```
Comment (DavidBoike) [inline src/CommandLineTests/NServiceBus.Transport.AzureServiceBus.CommandLine.Tests.csproj:4]: Did you mean to revert this?

File: src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj (hunk 1 above)
Comment (kbaley) [reply src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj:4]: No I didn't. Somewhere along the line, a backup file got created for this and the related test project files somehow. I'm guessing this is a consequence of that.

**Review by DavidBoike — APPROVED** [review]

**Review by DavidBoike — DISMISSED** [review]
Still needs netcoreapp6.0 to be fixed

**Review by bording — APPROVED** [review]

**Pull request merged**